### PR TITLE
Update pyOpenSSL stubs to 22.1.*

### PR DESCRIPTION
**Pull request by srittau**
Cf.  #8810

### CODE REVIEW

**Comment by srittau** [comment]
I *assume* that the `errors` argument should be `list[str]`, because the only initialization is here: https://github.com/pyca/pyopenssl/blob/7abe40cbff2b30f6b5d59b5a1106b7a6a237df86/src/OpenSSL/crypto.py#L1876-L1899

But I'm not 100% sure what those methods return. (Could be `bytes`.)

**Comment by srittau** [comment]
Actually it seems that `errors` is initialized with [int, int, str]. Changed `Incomplete` to `Any`.

File: stubs/pyOpenSSL/OpenSSL/crypto.pyi [hunk 1]
```diff
@@ -143,8 +143,9 @@ class X509StoreContext:
     def verify_certificate(self) -> None: ...
 
 class X509StoreContextError(Exception):
+    errors: list[Any]
     certificate: X509
-    def __init__(self, message: str | bytes, certificate: X509) -> None: ...
+    def __init__(self, message: str | bytes, errors: list[Any], certificate: X509) -> None: ...
```
Comment (JelleZijlstra) [inline stubs/pyOpenSSL/OpenSSL/crypto.pyi:148]: OpenSSL's inline annotations have `message` as just `str`. Not sure where we got `bytes` from. That's a separate issue from this PR though.

**Review by JelleZijlstra — APPROVED** [review]

File: stubs/pyOpenSSL/OpenSSL/crypto.pyi (hunk 1 above)
Comment (srittau) [reply stubs/pyOpenSSL/OpenSSL/crypto.pyi]: I changed it while I'm here.

**Pull request merged**